SlaveWAMPServer.js

```diff
@@ -145,7 +144,14 @@ class SlaveWAMPServer extends WAMPServer {
 			callback('RPC response timeout exceeded');
 			this.deleteCall(request);
 		}, this.internalRequestsTimeoutMs);
-		return setWith(this.interProcessRPC, `${request.socketId}.${request.procedure}.${request.signature}`, { callback, requestTimeout }, Object);
+
+		Object.assign(this.interProcessRPC, {
```

Comment:
What was the reason for using `setWith` and customizer `Object` rather than `set`? Why is the customizer no longer necessary?

Reply:
I cannot explain now why, hopefully the feature is not used anymore :)